### DISCUSSION TITLE
agent in k8s can't connect oap when enable mtls

### DISCUSSION THREAD

**Discussion by sruby** [apache/skywalking#8997] [Q&A] [answered]
Agent in k8s can't connect oap when enable mtls, but local agent(use same client and ca certificate) can connect oap normally.

Agent error log:
`
verification of certificate failed
No subject alternative names present
`

Client application env:

env: 
- name: TZ 
  value: "Asia/Shanghai" 
- name: SW_AGENT_COLLECTOR_BACKEND_SERVICES 
  value: skywalking-oap.skywalking-ns:11801 
- name: SW_AGENT_NAME 
   value: skywalking-demo 
- name: SW_AGENT_NAMESPACE 
  value: skywalking-ns 
- name: SW_AGENT_FORCE_TLS 
  value: "true" 
- name: SW_AGENT_SSL_TRUSTED_CA_PATH 
  value: /skywalking/agent/ca/ca.crt 
- name: SW_AGENT_SSL_KEY_PATH 
  value: /skywalking/agent/ca/client.pem 
- name: SW_AGENT_SSL_CERT_CHAIN_PATH 
  value: /skywalking/agent/ca/client.crt

Certificate created using the official script, server_cn=skywalking-oap.skywalking-ns .

How should I troubleshoot this problem?
Thanks.

**Comment by sruby** [comment, marked as answer]
Misconfiguration caused, please ignore this question.

**Comment by Superskyyy** [comment]
> Misconfiguration caused, please ignore this question.

Glad to see problem solved. What's better is if you could please share the way you solved it so others in the future can benefit from this. :)
